nengo_loihi/tests/test_learning.py

```diff
@@ -291,13 +334,13 @@ def test_pes_deterministic(request, Simulator, seed, allclose):
 # TODO: Revisit when we can blacklist boards
 @pytest.mark.xfail
```

Comment:
I looked into it, and thought it was still failing sometimes, but it might now just be because the learning tests occasionally time out, not the original reason that we xfailed for (which I think was because the test only passed on nahuku32).

Reply:
Looking at the last passing run, both `test_pes_deterministic` and `test_learning_seed` xfailed, so the original problem might still be there on nahuku32 (IIRC, it might have even been that the tests pass on one of the nahuku32 boards and not the other one).